### PR TITLE
Add Side by Side demo video generation

### PR DESCRIPTION
**Pull request by glaba13**
- Add script for  Side by Side demo video generation
- Add script for synchronous video recording
- Add script for video formatting
- Improved demo interface and add corresponding options
- Minor Cleanup of Code
- Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,3 +191,54 @@ For Unit Testing CppUtest framework is used.
...
+  The path should look like: *%PATH_TO_YOUR_DIR%\ffmpeg-%VERSION%\bin*
+  
+### Format Video:
+Formatting video makes start/end of the video detectable by adding a red frame at the end
+- In the *facemask-plugin/data* there is a script: **format.bat**
+- Put your sample video as input.mp4 and click on the format.bat
+- Or call the script from the shell 
+```console 
+./format.bat /path/to/the/video
+```
```
Comment (brankalakic) [inline README.md:216]: I was able to do this when running format.bat from the folder and after putting input.mp4. But I could not do it from command prompt, i get impossible to open for the file. Also, can you add a destination path when calling it from command line?

File: README.md [hunk 2]
```diff
@@ -192,3 +191,54 @@ For Unit Testing CppUtest framework is used.
...
+	* Recording and Rendering will take some time
+	
+### Generate Side By Side Video:
+
+* Go to Setting of facemask-plugin:
+    * Choose the before video
+	* Configure the before text for the video
+	* Choose the after video
+	* Configure the after text for the video
+    * Push the Generate button
```
Comment (brankalakic) [inline README.md:247]: When I push the generate button it freezes.

Comment (glaba13) [reply README.md:247]: It takes some time, but you should see the script which is running.
Did you copy the data folder with DLL?

File: README.md (hunk 1 above)
Comment (glaba13) [reply README.md:216]: - Fixed it, was a problem with windows standard command line (working in bash I have)
- Added output file parameter

File: README.md (hunk 2 above)
Comment (brankalakic) [reply README.md:247]: That helped.

File: README.md [hunk 3]
```diff
@@ -192,3 +191,58 @@ For Unit Testing CppUtest framework is used.
...
+- Or call the script from the shell 
+```console 
+./sidebyside.bat /path/to/the/beforeVideo /path/to/the/afterVideo before-text after-text /path/to/output/file
+```
+
```
Comment (brankalakic) [inline README.md:252]: Before text is always "Before", even when I make the change in command line or in slobs. After text works

File: README.md [hunk 4]
```diff
@@ -192,3 +191,58 @@ For Unit Testing CppUtest framework is used.
...
+- ##### Manually
+  - Start Steramlabs-OBS recorder *(don't forget to change output format (in settings/output) to mp4)*
+  - Start and End recording as you see **red frame** appears
+- ##### Automotive
+  * Add a filter: Face Mask Plugin
+  * Go to Setting of facemask-plugin:
+    * Set the **Demo Mode Folder** Path (where the output video should be generated)
+	* Turn On **Record Video**
+	* Recording and Rendering will take some time
+
```
Comment (brankalakic) [inline README.md:239]: When you generate the frames, could you put them in a tmp folder inside of the Demo Folder and then delete tmp folder in the end the way you delete the frames right now? Also, can you name the videos based on the date/time like the ones that are created when recording using REC button

**Comment by glaba13** [comment]
Added temp folder,
Added Timestamp as a video name

**Pull request merged**